Source/WebKit/NetworkProcess/storage/NetworkStorageManager.cpp

```diff
@@ -484,7 +484,7 @@ void NetworkStorageManager::performEviction(HashMap<WebCore::SecurityOriginData,
     assertIsCurrent(workQueue());
 
     m_isEvictionScheduled = false;
-    if (!m_totalQuota || !m_totalUsage || *m_totalUsage <= m_totalQuota)
+    if (!m_totalQuota || !m_totalUsage || *m_totalUsage <= *m_totalQuota)
```

Comment:
At this point `m_totalQuota` should already be initialized (otherwise eviction should not be scheduled), so let's change it to 
`if (!m_totalUsage || *m_totalUsage <= *m_totalQuota)` and add an assert for `m_totalQuota`.